kitsune/search/search.py

```diff
@@ -323,7 +323,7 @@ class CompoundSearch(SumoSearch):
     _children: list[SumoSearch] = dfield(default_factory=list, init=False)
     _parse_query: bool = True
 
-    @property
+    @property  # type: ignore
```

Comment:
Even though this code works perfectly, `mypy` complains that `Dataclass attribute may only be overridden by another attribute`, so that's why the `# type: ignore` is here.